pandas/_libs/tslibs/tzconversion.pyx

```diff
@@ -457,7 +457,7 @@ cdef int64_t[:] _tz_convert_from_utc(int64_t[:] vals, tzinfo tz):
         str typ
 
     if is_utc(tz):
-        converted = vals
+        return vals
```

Comment:
doesnt need to be this PR, but might be convenient: i think this should probably return a copy, not the original array.

Reply:
I agree this should be a copy, can you update @lidavidm otherwise lgtm. pls ping on green.